firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -32,8 +32,10 @@ class ActivityStreamTest: BaseTestCase {
                                LaunchArguments.SkipContextualHints,
                                LaunchArguments.DisableAnimations]
         }
-        launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
-        launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
+        // These launchArguments are commented out so that the shortcuts are shown after
+        // the homepage redesign.
+        // launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
+        // launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
```

Comment:
@cyndichin Let me know the issue number so that I can refer to the investigation here.

@isabelrios @dragosb01 This PR contains just the changes related to the homepage redesign. Should we merge the changes to reduce some noise in the full functional tests?

Reply:
https://mozilla-hub.atlassian.net/browse/FXIOS-11668